modules/tinymce/src/themes/silver/main/ts/ui/general/Button.ts

```diff
@@ -25,7 +24,9 @@ import { formActionEvent, formCancelEvent, formSubmitEvent } from './FormEvents'
 type Behaviours = Behaviour.NamedConfiguredBehaviour<any, any, any>[];
 type AlloyButtonSpec = Parameters<typeof AlloyButton['sketch']>[0];
 
-type ButtonSpec = Omit<Dialog.Button, 'type'>;
+type ButtonSpec = Omit<Dialog.Button, 'type'> & {
+  readonly?: boolean;
+};
```

Comment:
Oh this relates to the dialog button comment I made earlier? That's a weird way to define a button, and definitely makes things more complicated 😬

Reply:
Removed that in the Dialog.button.